src/Neo.VM/Unsafe.cs

```diff
@@ -38,5 +41,16 @@ public static bool NotZero(ReadOnlySpan<byte> x)
             }
             return false;
         }
+
+        /// <summary>
+        /// Get 64-bit hash code for a byte array
+        /// </summary>
+        /// <param name="span">Span</param>
+        /// <returns></returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ulong HashBytes(ReadOnlySpan<byte> span)
+        {
+            return XxHash3.HashToUInt64(span, HashMagicNumber);
```

Comment:
This will overflow `int`? Does it return `8-byte` integer all the time? If so then use what I had.

Reply:
It's faster, and official, we can convert this ulong to a new int

Reply:
could use `XxHash32.HashToUInt32();

Reply:
> could use `XxHash32.HashToUInt32()`

`Xxhash3` is faster then `XxHash32`

Reply:
```csharp
public uint ConvertUlongToUint(ulong input)
{
    uint lowerBits = (uint)(input & 0xFFFFFFFF);
    uint upperBits = (uint)((input >> 32) & 0xFFFFFFFF);
    return lowerBits ^ upperBits;
}
```

Reply:
fixed please recheck

Reply:
> > could use `XxHash32.HashToUInt32()`
> 
> `Xxhash3` is faster then `XxHash32`

it is more suitable for current computers 64 bits architetures